src/ja/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: home
+hero:
+  name: "The JavaScript Oxidation Compiler"
+  tagline: Rust製JavaScriptツール群
+  image:
+    src: https://raw.githubusercontent.com/oxc-project/oxc-assets/main/logo-round.png
+    alt: The JavaScript Oxidation Compiler
+  actions:
+    - theme: brand
+      text: はじめる
+      link: /ja/docs/guide/introduction
+    - theme: alt
+      text: GitHubで見る
+      link: https://github.com/oxc-project/oxc
+features:
+  - title: パーサ ✅
```

Comment:
I feel that it might be better not to translate these terms too much into Japanese, and to keep words like 'Parser', 'Linter', etc., as they are. It seems like Biome also uses 'format', 'lint' as they are. What do you think?

https://biomejs.dev/ja/

Reply:
I'm fine with it, rather than using Katakana words.
Actually, I wondered if I should call it as `パーサ` or `パーサー` 😅  
Using English will simplify the situation 👍

Reply:
Yeeeeees. I always think the same.